### DISCUSSION TITLE
Welcome to mtf-rwanda-analysis Discussions!

### DISCUSSION THREAD

**Discussion by HEDERA-PLATFORM** [HEDERA-PLATFORM/mtf-rwanda-analysis#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HEDERA-PLATFORM** [comment]
## My first comment in this discussion

Would it make sense to use this? @alfonsocaiazzo 
* maybe not only in this repo but also for the other
* this has the purpose to inform that this is now available
* we can decide if we want to use it

**Reply by alfonsocaiazzo** [reply]
Yes, good idea! We can use it for general topics, and create issues only in concrete cases.

**Reply by HEDERA-PLATFORM** [reply]
Agree. Did you get an email informing you that I have tagged you?
